model/Core/Vocabularies/AnnotationType.md

```diff
@@ -4,18 +4,17 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+Specifies the type of an annotation.
 
 ## Description
 
-TODO
+AnnotationType specifies the type of an annotation.
 
 ## Metadata
 
 - name: AnnotationType
 
 ## Entries
 
-- other: TODOdescription
-- review: TODOdescription
-
+- other: Used to store extra information about an Element during creation.
```

Comment:
Although this generally follows the 2.3 documentation, the other type may also be used after the creation.  Suggest softening this a bit like: 

`Used to store extra information about an Element which is not part of a Review (e.g. extra information provided during the creation of the Element).`

Reply:
done